dataframe-jdbc/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/readJdbc.kt

```diff
@@ -831,6 +846,25 @@ private fun fetchAndConvertDataFromResultSet(
     return dataFrame
 }
 
+private fun handleArrayValues(values: MutableList<Any?>): List<Any> {
+    // Intermediate variable for the first mapping
+    val sqlArrays = values.map {
+        (it as java.sql.Array).array ?: emptyArray<Any>()
+    }
+
+    // Determine the type based on the first element
+    val firstArray = sqlArrays.firstOrNull()
+    val firstElementType = firstArray?.javaClass?.componentType?.kotlin ?: Any::class
```

Comment:
can we be sure this never breaks? Say the array consists of Longs but the first element is "123" which can be read as an integer, or even a short, will this break things? Or are the types decided elsewhere based on the entire contents of the array?